firebase-auth.html

```diff
@@ -136,6 +136,20 @@
         },
 
         /**
+         * Authenticates a Firebase client using an oauth id_token.
+         *
+         * @return {Promise} Promise that handles success and failure.
+         */
+        signInWithCredential: function(credential) {
+          if (!this.auth) {
+            return Promise.reject('No app configured for firebase-auth!');
+          }
+
+          return this._handleSignIn(this.auth.signInWithCredential(credential));
+        },
+
+
+        /**
```

Comment:
That's weird by I still see that new line. Also, i'm not a maintainer of the project, so you have to wait for maintainers to come merge it, if they decide this is needed feature.